README.md

```diff
@@ -204,16 +133,18 @@ dH/dt = dHdt      (1c)
 ```
 The [`diffusion_1D_expl.jl`](scripts/diffusion_1D_expl.jl) code implements an iterative and explicit solution of eq. (1) for an initial Gaussian profile:
 ```md
-H = exp(-(x-lx/2.0)^2)
+H0 = exp(-(x-lx/2.0)^2)
```

Comment:
This goes with the slight code-change I did.

Reply:
This is good with `H0`

Reply:
From my perspective you can merge @mauro3.